cms/templates/maintenance/_announcement_index.html

```diff
@@ -0,0 +1,49 @@
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<div class="studio-view maintenance-form">
+    <div class="form-create">
+        <div class="announcement-container">
+            % for announcement in announcement_list:
+                <div class="announcement-item">
+                    <div class="announcement-content">
+                        ${announcement.content | n} ## xss-lint: disable=mako-invalid-html-filter
```

Comment:
Would like a visible marker to show which announcements are disabled, and which are enabled, since this will get harder to manage as new announcements are added.

Maybe also if you sort them to show the active announcements first, in the order they'll be displayed on the LMS?

Reply:
I added a text that says "Active" or "Inactive", with a checkmark or an empty box. Does that seem like a good indicator? I wasn't sure exactly where or how to indicate this.

I've added a sort by active status on the queryset, so the inactive announcements will show up last.

Reply:
Beaut! 👍